src/System.IO/tests/System.IO.Tests.csproj

```diff
@@ -59,6 +59,10 @@
     <Compile Include="$(CommonTestPath)\System\IO\WrappedMemoryStream.cs">
       <Link>Common\System\IO\WrappedMemoryStream.cs</Link>
     </Compile>
+    <Compile Include="TextReader\TextReaderImpl.cs" />
+    <Compile Include="TextReader\TextReaderTests.cs" />
+    <Compile Include="TextWriter\TextWriterImpl.cs" />
+    <Compile Include="TextWriter\TextWriterTests.cs" />
```

Comment:
The tests for the new APIs should be separated out from tests for the existing ones, such that the file containing the new ones can be conditioned to only build when those APIs exist, e.g. ` Condition="'$(TargetGroup)' == 'netcoreapp'" `.  That'll help with the build failures.